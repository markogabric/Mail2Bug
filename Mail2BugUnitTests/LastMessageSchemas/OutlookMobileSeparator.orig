<html><head></head>
<body>
<div>
  <div>
    <div style="customStyle">Message content from iPhone</div>
  </div>
  <div><br /></div>
  <div class="ms-outlook-ios-signature"></div>
</div>
<hr style="display:inline-block;width:98%" tabindex="-1" />
<div>
    Something after the containing div
</div>
</body>
</html>