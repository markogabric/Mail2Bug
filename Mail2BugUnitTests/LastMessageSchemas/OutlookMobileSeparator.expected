<html><head></head>
<body>
<div>
  <div>
    <div style="customStyle">Message content from iPhone</div>
  </div>
  <div><br></div>
  <div class="ms-outlook-ios-signature"></div>
</div>
</body></html>